programmer-skills/start-to-develop-js.md

```diff
@@ -159,11 +158,11 @@ there:
 ![alert with your name](/images/brackets_alert.png)
 
 This is an easy way to start trying JavaScript snippets, but would be
-better if we split our html with JavaScript in two files. One for html,
-and other for JavaScript code.
+better if we split our html with JavaScript into two files. One for html,
+and another for JavaScript code.
 
 - Create a JavaScript file called `myscript.js` and save it in the same
-  folder where you saved your html. This is what you must to put there:
+  folder where you saved your html. This is what you must put there:
```

Comment:
"Enter the following lines of code into the file" >>> "This is what you must put there:"